docs/source/usage.rst

```diff
@@ -34,7 +34,7 @@ There are a few command line options available::
       --help          Show this message and exit.
 
```

Comment:
The note about the added feature should also be in an indented block, see [the sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded) for an example. Otherwise there is no visual indication **what** was actually added in that version.

Reply:
Done.